screens.py

```diff
@@ -0,0 +1,112 @@
+import curses
+import time
+from get_words import get_random_word
+from calculate_score import cal_accuracy_score, cal_time_score
+
+def start_screen(stdscr):
+    """Displays initial screen of typing tester"""
+
+    stdscr.clear()
+    stdscr.addstr(2,0,"Typing Tester", curses.color_pair(1))
+    stdscr.addstr(3,0,"Press any key to start.....")
+    stdscr.refresh()
+    stdscr.getkey()
+    typing_tester_screen(stdscr) # Test Screen where user input is evalauted
+
+def typing_tester_screen(stdscr):
```

Comment:
What does this function return?

Reply:
It's a void function that displays the screen where user performs typing test